nmostesting/mocks/Node.py

```diff
@@ -211,6 +212,16 @@ def parse_sdp(self, sdp_data):
                 sdp_param_leg["source_ip"] = filter_line.group(2)
             else:
                 sdp_param_leg["source_ip"] = None
+
+            format_line = re.search(r"a=fmtp:(\S*\s*)(.*)", sdp_data)
+
+            if format_line and format_line.group(2):
+                #  Handle parameter keys that have no value, e.g. 'interlace' and set the value to 'True'
+                sdp_param_leg["format"] = {key: ''.join(value) if len(value) > 0 else 'True'
```

Comment:
The thorough mix of single quotes and double quotes is starting to get to me. 😅

Reply:
Yes, I know - I've now tried to make the use of quotes a *little* more consistent